src/Config_Command.php

```diff
@@ -349,6 +349,255 @@ private static function get_wp_config_vars() {
 		return array_merge( $wp_config_vars, $wp_config_constants, $wp_config_includes_array );
 	}
 
+	/**
+	 * Sets the value of a specific variable or constant defined in
+	 * wp-config.php file.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
```

Comment:
Think should use the more natural `name` than `key`. Also missed this in the previous get review.